### ISSUE TITLE
How do i use customizationObject in the chat widget in the Chat.js

### ISSUE THREAD

**Issue by Richab2310** [amazon-connect/amazon-connect-chat-interface#269]
I need to add a dropdown in the header in the chat widget. i can do that with customizationObject in the script, but how do i add this in the chat.js file. Is there any way i can add a dropdown in the header of the chat widget?
<img width="325" alt="image" src="https://github.com/user-attachments/assets/8e812672-f1ad-4739-8492-dda30d9dcbc6">

**Comment by mrajatttt** [comment]
Hi @Richab2310, Let me see if i understand it correctly. You are aware of how to enable dropdown using widget snippet but you want to implement dropdown using code rather than using snippet?

**Comment by Richab2310** [comment]
Yes, correct. I know how to do it from the widget snippet. i need it to implement from the code mainly from the Chat.js.

**Comment by mrajatttt** [comment]
This is a part of Chat Interface UI so it cannot be controlled at Chat.js. May I ask if you are using your own UI or out-of-the box?

**Comment by mrajatttt** [comment]
Have not seen any activity here since 2 months, Closing it for now. Feel free to reopen.
